### PR TITLE
fix: swap successful redirect

### LINKED ISSUE #11226: Swap Landing follow up
> * after success app should take you to your profile / swaps / outgoing - currently it redirects you to /swaps

sure, let's do it in a follow up since the code has been tested and also reviewed

_Originally posted by @hassnian in https://github.com/kodadot/nft-gallery/issues/11163#issuecomment-2527727227_

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [Koda - Generative Art Marketplace](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix

## Needs QA check

- @kodadot/qa-guild please review

## Context

- [x] Closes #11226

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Comment by hassnian** [comment]
notification text got a little long, let's use this




it's already implemented in `MakeOfferModal.vue` -> `showIndexerDelayMessage`

**Comment by Jarsen136** [comment]
> notification text got a little long, let's use this
> 
> 
> it's already implemented in `MakeOfferModal.vue` -> `showIndexerDelayMessage`
> 
> or implement that prop in a normal transaction notification

updated

image

File: components/swap/review.vue [hunk 1]
```diff
@@ -134,14 +135,16 @@ const submit = () => {
...
     urlPrefix: urlPrefix.value,
-    successMessage: $i18n.t('swap.created'),
   })
 }
 
-watchEffect(async () => {
+watch([status, blockNumber], () => {
   if (status.value === TransactionStatus.Finalized && blockNumber.value) {
+    successMessage($i18n.t('swap.created'), {
+      footer: computed(() => ({ icon: 'circle-info', label: $i18n.t('general.updateOnWebsiteSoon') })),
```
Comment (hassnian) [inline components/swap/review.vue:144]: would make the footer `MaybeRef`

```suggestion
      footer: { icon: 'circle-info', label: $i18n.t('general.updateOnWebsiteSoon') },
```

Comment (hassnian) [inline components/swap/review.vue:144]: would make the footer `MaybeRef`

```suggestion
      footer: { icon: 'circle-info', label: $i18n.t('general.updateOnWebsiteSoon') },
```

File: utils/notification.ts [hunk 2]
```diff
@@ -120,11 +120,12 @@ export const infoMessage = (
   })
 }
 
-export const successMessage = message =>
+export const successMessage = (message: string, extraConfig?: { footer?: Ref<Omit<NotificationAction, 'url'> | undefined> }) =>
```
Comment (hassnian) [inline utils/notification.ts:123]: if you could make a `NotificationFooter` type

```suggestion
export const successMessage = (message: string, extraConfig?: { footer?: NotificationFooter | undefined> }) =>
```

**Review by hassnian — COMMENTED** [review]
otherwise LGTM

File: components/swap/review.vue (hunk 1 above)
Comment (Jarsen136) [reply components/swap/review.vue]: updated

File: utils/notification.ts (hunk 2 above)
Comment (Jarsen136) [reply utils/notification.ts]: updated

**Review by hassnian — APPROVED** [review]

**Review by preschian — APPROVED** [review]

**Pull request merged**